src/locales/pt-br/translation.json

```diff
@@ -0,0 +1,181 @@
+  "Alert Custom Detail Column Definitions Placeholder": "Marcador de Coluna de Definições dos Detalhes Personalizados do Alerta",
+  "and their alerts have been merged onto incident": "e os seus alertas foram combinados no incidente",
+  "Assignees": "Detentores",
+  "Auto Accept Incident Query": "Consulta de incidente de aceitação automática",
+  "Auto Refresh Interval (mins)": "Intervalo de atualização automática (min)",
+  "Available": "Disponível",
+  "because they have already been suppressed or resolved": "porque já foram eliminados ou resolvidos",
+  "Cancel": "Cancelar",
+  "characters remaining": "caracteres restantes",
+  "Class": "Classe",
+  "Clear Local Cache": "Limpar Cache Local",
+  "Column Selector": "Seletor de Coluna",
+  "Component": "Componente",
+  "Connect using PagerDuty OAuth to use this app": "Conectar usando PagerDuty OAuth para usar esta aplicação",
+  "Connected": "Conectado",
+  "Connecting": "Conectando",
+  "Continue": "Continuar",
+  "Created At": "Criado Em",
+  "critical": "urgente",
+  "Current query parameters match X incidents": "Os parâmetros de consulta atuais coincidem com {{totalIncidentsFromQuery}} incidentes",
+  "Current subdomain does not have the correct ability to use PagerDuty Live": "O subdomínio atual não tem a capacidade correta para usar PagerDuty Live",
+  "Custom": "Personalizado",
+  "Custom Incident Action": "Ação de Incidente Personalizada",
+  "Decline": "Recusar",
+  "Default Since Date Lookback": "Retrospetiva a partir de Data Predefinida",
+  "Description": "Descrição",
+  "Disclaimer & License": "Aviso Legal & Licença",
+  "error": "erro",
+  "Escalate": "Escalar",
+  "Escalation Policy": "Política de Escalação",
+  "External References": "Referências Externas",
+  "External Systems": "Sistemas Externos",
+  "Fetching Alerts": "Buscando Alertas",
+  "Fetching Notes": "Buscando Anotações",
+  "Filter available": "Filtro disponível",
+  "Filter selected": "Filtro selecionado",
+  "Filters": "Filtros",
+  "Group": "Grupo",
+  "have been": "foram",
+  "have been manually escalated to level": "foram escalados manualmente para o nível",
+  "have been reassigned to": "foram reatribuídos a",
+  "have been updated with a note": "foram atualizados com uma anotação",
+  "have been updated with priority": "foram atualizados com a prioridade",
+  "High": "Alta",
+  "hour": "hora",
+  "hours": "horas",
+  "I agree to the disclaimer and license above": "Aceito o aviso legal e a licença acima",
+  "incident": "incidente",
+  "Incident": "Incidente",
+  "Incident ID": "ID do Incidente",
+  "Incident Key": "Chave do Incidente",
+  "Incident Table": "Tabela de Incidentes",
+  "Incidents": "Incidentes",
+  "info": "informação",
+  "Invalid JSON Path": "Caminho JSON inválido",
+  "Last Status Change At": "Última Alteração de Estado às",
+  "Last Status Change By": "Última Alteração de Estado por",
+  "Latest Note": "Última Anotação",
+  "Live Incidents Console": "Consola de Incidentes ao Vivo",
```

Comment:
I think that in Brazil is more common to use this term in English, so probably we should change to "Console" instead of "Consola"

Reply:
```suggestion
  "Live Incidents Console": "Console de Incidentes ao Vivo",
```